src/__screenshot_tests__/stacking-group-screenshot-test.tsx

```diff
@@ -0,0 +1,24 @@
+import {openStoryPage, screen} from '../test-utils';
+
+test.each`
+    size    | type        | stacked  | inverse
+    ${'64'} | ${'image'}  | ${false} | ${false}
+    ${'64'} | ${'avatar'} | ${false} | ${false}
+    ${'64'} | ${'image'}  | ${true}  | ${false}
+    ${'64'} | ${'avatar'} | ${true}  | ${false}
+    ${'64'} | ${'image'}  | ${false} | ${true}
+    ${'64'} | ${'avatar'} | ${false} | ${true}
+    ${'64'} | ${'image'}  | ${true}  | ${true}
+    ${'64'} | ${'avatar'} | ${true}  | ${true}
```

Comment:
why is 64 a param in the test? it's always 64

Reply:
In the Screenshot tests, the default was 64 to avoid repeating the prints for each of the screen sizes (24, 32 and 64) with variations but due to the switch to accept any component now the only size change available is that of the element that shows the surplus number, which in the story was configured so that the same value is used in the image component.